docs/workflow/blocks/mapping.rst

```diff
@@ -53,9 +53,39 @@ read from *filter*. Reading from  *state.global.workflowCloud.listWorkflows.filt
 Handy JMESPath patterns
 -----------------------
 
+Using string as value
+^^^^^^^^^^^^^^^^^^^^
+When manually creating data, a string will be passed as value.
```

Comment:
```suggestion
Data can be made manually using a string value.
```

Reply:
I rewrote it in a different way, let me know if is understable